keyleak/site_scanner.py

```
@@ -29,165 +39,309 @@
+                try:
+                    context = browser.new_context(viewport={"width": 1280, "height": 1024})
+                    page = context.new_page()
+                    page.set_default_timeout(15000)
+                    page.goto(url, wait_until="domcontentloaded")
+                    links = page.evaluate(
+                        "() => Array.from(document.querySelectorAll('a[href]'))"
+                        ".map(a => a.href).filter(h => h.startsWith('http'))"
+                    )
+                    context.close()
+                except Exception:
+                    continue
```

Ruff (0.15.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)